crates/ruff_server/src/session/index.rs

```diff
@@ -187,6 +187,21 @@ impl Index {
         Self::register_workspace_settings(&mut self.settings, url, None, global_settings)
     }
 
+    pub(super) fn count_documents(&self) -> usize {
+        self.documents.len()
+    }
+
+    pub(super) fn count_workspaces(&self) -> usize {
+        self.settings.len()
+    }
```

Comment:
Nit: The method names suggest that the implementation counts the documents and workspaces but that isn't the case, it just takes the len. Maybe `num_documents()` and `num_workspaces`?